packages/docz-core/src/Entries.tsx

```diff
@@ -114,6 +114,9 @@ export class Entries {
     const entries = await Promise.all(files.filter(isEntry).map(createEntry))
 
     for (const entry of entries) {
+      if (process.platform === "win32") {
+        entry.filepath = entry.filepath.split('\\').join('/')
```

Comment:
I think you can use `path.normalize` https://nodejs.org/api/path.html#path_path_normalize_path

❯ node -p "require('path').win32.normalize('/Users/test')"
\Users\test